src/Mailhog_Command.php

```diff
@@ -32,6 +31,8 @@ class Mailhog_Command extends EE_Command {
 	 */
 	public function up( $args, $assoc_args ) {
 
+		\EE\Auth\Utils\init_global_admin_tools_auth();
```

Comment:
How do you know this function will always be accessible? There is no [composer dependency](https://github.com/EasyEngine/mailhog-command/blob/develop/composer.json) also. Please check if `method_exists( '\EE\Auth\Utils\init_global_admin_tools_auth' )` then procced next. 

Since this is a required step you can throw error saying that auth package is required when this method is not exists.